src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -67,11 +67,11 @@ public function collect(Request $request, Response $response, \Exception $except
             'env' => isset($this->kernel) ? $this->kernel->getEnvironment() : 'n/a',
             'debug' => isset($this->kernel) ? $this->kernel->isDebug() : 'n/a',
             'php_version' => PHP_VERSION,
+            'php_architecture' => PHP_INT_SIZE * 8,
+            'php_locale' => \Locale::getDefault() ?: 'n/a',
```

Comment:
I would relabel it. The Intl locale is more useful than the PHP locale in Symfony, as our own code does not deal with the PHP locale (it would be useful to display it if you were using ext-gettext to handle translations)